src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Client/Properties/launchSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "useWebAssemblyDebugging": true
+}
```

Comment:
@ajaybhargavb I don't think we need it on the client template.

Reply:
We need this config for Blazor standalone apps as well.